manifests/pnp4nagios/install.pp

```diff
@@ -0,0 +1,10 @@
+class icinga2::pnp4nagios::install inherits icinga2::pnp4nagios {
+  package {$package_name:
+    ensure => $package_ensure,
+  }
+
+  package { ['wget','unzip']:
```

Comment:
Probably wanna be careful managing generic packages like this here. At least provide a parameter $manage_dependencies or something.